.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -43,6 +43,7 @@ body:
         - All or N/A
         - WCOSS2
         - Hera
+        - Ursa
         - Gaea-C5
```

Lets remove `Gaea-C5` 
```suggestion
        - Ursa
```